py3-jupyterhub-idle-culler.yaml

```diff
@@ -0,0 +1,38 @@
+# Generated from https://pypi.org/project/jupyterhub-idle-culler/
+package:
+  name: py3-jupyterhub-idle-culler
+  version: 1.2.1
+  epoch: 0
+  copyright:
+    - license: 3-BSD
```

Comment:
Is this correct? Doesn't look like an abbreviation we've used before and I don't see listed in: https://opensource.org